src/api/app/controllers/webui/request_controller.rb

```diff
@@ -488,6 +490,10 @@ def set_action_id
     @action_id = params[:request_action_id] || @supported_actions.first&.id || @actions.first.id
   end
 
+  def set_bs_request_action
+    @bs_request_action = BsRequestAction.find(@action_id)
```

Comment:
This will allow me to load *any* BsRequestAction onto the page that I want just by passing the parameter...

Reply:
Addressed in the last changes